.github/workflows/auto-merge.yml

```
@@ -0,0 +1,124 @@
+              `### Failed Checks:\n${checkStatus.failedChecks.map(c => `- ${c}`).join('\n')}` : 
+              ''
+            }
+            
+            Please resolve these issues for auto-merge to proceed.`;
+            
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.payload.pull_request.number,
+              body: comment
+            });
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)